Documentation/operations/upgrade.rst

```diff
@@ -616,7 +636,7 @@ Upgrading from >=1.7.0 to 1.8.y
       helm template cilium \\
       --namespace=kube-system \\
       ...
-      --set global.bpf.natMax=841429
+      --set bpf.natMax=841429
```

Comment:
this should not be modified since this section referes to upgrade from 1.7 to 1.8, we aren't changing the helm chart of 1.8 branch.

Reply:
fixed

Reply:
not addressed